AutomatticTracks/src/main/java/com/automattic/android/tracks/DeviceInformation.java

```diff
@@ -125,6 +125,7 @@ public DeviceInformation(Context context) {
             mImmutableDeviceInfoJSON.put("app_name", getAppName());
             mImmutableDeviceInfoJSON.put("app_version", getAppVersionName());
             mImmutableDeviceInfoJSON.put("app_version_code", Integer.toString(getAppVersionCode()));
+            mImmutableDeviceInfoJSON.put("language", getDeviceLanguage());
```

Comment:
some fields here are prefixed with `app_`, should we name that field `device_language` so it's obvious that's not the app language?